### ISSUE TITLE
Add properties for primary and secondary connecting strings to `azure.cosmosdb.Account`

### ISSUE THREAD

**Issue by CraftyFella** [pulumi/pulumi-azure#649]
Hi,

I'm using the `azure.cosmosdb.Account` to create a cosmos DB in azure. It's working as expected however to get the connecting string out is a bit of a pain. 

Often on resources created with pulumi I can access the 

primaryConnectingString

with this resource I  have to build it up using the following code:

```
export const buildCosmosTableStorageConnectingString = (account: azure.cosmosdb.Account) => {
  pulumi.interpolate `DefaultEndpointsProtocol=https;AccountName=${account.name};AccountKey=${account.primaryMasterKey};TableEndpoint=https://${account.name}.table.cosmos.azure.com:443/;`
}
```

Are you planning or would you consider exposing the primary and secondary Connection strings so I can avoid the above?

<img width="254" alt="image" src="https://user-images.githubusercontent.com/685872/89025129-3206af80-d31e-11ea-9cdb-299dbfeaa545.png">

**Comment by mikhailshilkov** [comment]
Hey @CraftyFella thanks for your request.
Doesn't `connectionStrings` output property work for you? https://www.pulumi.com/docs/reference/pkg/azure/cosmosdb/account/#connectionstrings_nodejs

**Comment by CraftyFella** [comment]
yeah it works.. but it's a list and I can do `.connectionStrings[4]` but I'm not sure if I can guarentee that the 5th element in the list will always be the primary connection string. The UI shows the primaryConnection string, so just wondering if there was a reason not to add it?

**Comment by mikhailshilkov** [comment]
We derive this API from the Terraform provider, so any change like this would likely need to be contributed upstream
